ckan/model/activity.py

```diff
@@ -170,14 +172,18 @@ def user_activity_list(user_id, limit, offset):
 
     return _activities_at_offset(q, limit, offset)
 
+def _to_list(vals):
+    if isinstance(vals, (list, tuple)):
+        return vals
+    return [vals]
```

Comment:
@EricSoroos can you add a small docstring to this function? What it does is clear, however it is not clear why we need it. I found this internal functions hard to maintain in the future if it is not clear why are there.

Looks that sometimes `vals` it is a tuple, others a list, and sometimes just a value. Where you able to identify why is that case? The variable names refers to `user_id` or `package_id` which makes you think it is singular but I guess someone is passing lists to it?

Reply:
It's probably just a list, but I don't want to be returning [(vals,)].

It was user_id or package_id, but I've overloaded those items to potentially have a list instead to minimize the changes here, so we're doing a  `field in (values)`, but those values could still be singular string uids.